README.md

```diff
@@ -76,15 +76,17 @@ export const GALLERY_ITEMS = [
 // your-gallery.js
 
 import { GALLERY_ITEMS } from "./some-data"
-import { Gallery, GalleryMain, GalleryNav, GalleryPagination } from "@wethegit/react-gallery"
+import { Gallery, GalleryMain, GalleryNav, GalleryPagination, GalleryItem } from "@wethegit/react-gallery"
 
 const YourGallery = () => {
   return (
     <Gallery items={GALLERY_ITEMS}>
 
       <GalleryMain
-        renderGalleryItem={({ item }) => (
-          <img src={item.image} alt={item.alt} />
+        renderGalleryItem={({ item, i, active }) => (
+          <GalleryItem key={i} i={i} active={active}>
```

Comment:
I think the prop here is called `index`, not `i`, right?

Reply:
Correct, fixed!